x/leverage/client/tests/suite.go

```diff
@@ -106,8 +106,6 @@ func (t testQuery) Run(s *IntegrationTestSuite) {
 		fmt.Sprintf("--%s=json", tmcli.OutputFlag),
 	}
 	out, err := clitestutil.ExecTestCLICmd(clientCtx, t.command, append(t.args, queryFlags...))
-	// TODO: enable err check https://github.com/umee-network/umee/issues/1351
-	// require.NoError(err, t.msg)
 
 	if t.expectErr {
 		require.Error(err, t.msg)
```

Comment:
The `require.NoError` line should not be run in `t.expectErr` tests. It is already present in the `else` block though.